backend/src/router_test.go

```diff
@@ -515,6 +515,30 @@ func (suite *RouterSuite) TestPatchProfile() {
 	}))
 }
 
+func (suite *RouterSuite) TestPutTutorsUsername() {
+	suite.Run("Success", manualSetupTest(func() {
+		hash, _ := bcrypt.GenerateFromPassword([]byte("Password"), bcrypt.DefaultCost)
```

Comment:
Using `POST /signup` should work without have to manually create the user.

Reply:
Added